src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -144,14 +141,12 @@ function assertManifestEndpoint(ep: ManifestEndpoint, id: string): void {
     scheduleTrigger: "object",
     taskQueueTrigger: "object",
     blockingTrigger: "object",
-    cpu: (cpu: backend.Endpoint["cpu"]) =>
-      cpu === null || typeof cpu === "number" || cpu === "gcf_gen1",
+    cpu: "Field<number>?",
```

Comment:
how does `Field<number>` correctly allow "gcf_gen1"?

Reply:
well, it can be an Expression<number>, which can be an arbitrary string because we don't have any kind of sanity checking on things purporting to be CEL at the type system level, which god just typing that out makes me feel dirty; yeah I'm going to make that explicit.

Reply:
Done.